pandas/core/arrays/string_.py

```diff
@@ -283,7 +283,6 @@ def __setitem__(self, key, value):
         super().__setitem__(key, value)
 
     def fillna(self, value=None, method=None, limit=None):
-        # TODO: validate dtype
         return super().fillna(value, method, limit)
```

Comment:
does this method need to be overriden here at all?

Reply:
it doesn't

Reply:
pushed that now